homeassistant/components/mqtt/schemas.py

```diff
@@ -0,0 +1,167 @@
+"""Schemas for MQTT discovery."""
```

Comment:
It would be nice to do the schema break out in a separate PR. It would really help with testing this so I can focus on making sure I'm profiling the changed code vs the existing code as I spent a bit more time digging around in the profile only to realize it was moved code and not new code.